tsl/src/continuous_aggs/create.c

```diff
@@ -519,8 +517,9 @@ get_timebucketfnoid()
 	Oid funcoid;
 	const char *funcname = TIMEBUCKETFN;
 	CatCList *catlist = SearchSysCacheList1(PROCNAMEARGSNSP, CStringGetDatum(funcname));
+	int i;
 
-	for (int i = 0; i < catlist->n_members; i++)
+	for (i = 0; i < catlist->n_members; i++)
```

Comment:
Can we figure out how to get the compiler to fail (either locally or on Travis) for this kind of thing? Sometimes it doesn't get caught and it causes problems when building on Debian/Ubuntu

Reply:
Well  coverity fails but not noisily. I can ask @JLockerman to take a look. But not for this PR.

Reply:
disable the warning everywhere 😜 
more seriously, the warning isn't widely available, I think it's only some versions of GCC that offer it.